src/vm/syncblk.cpp

```diff
@@ -3337,7 +3337,8 @@ bool AwareLock::Contention(INT32 timeOut)
                     bKeepGoing = false;
                     break;
                 }
-                
+
+#ifdef WINDOWS_AMD64_ABI
```

Comment:
This has nothing to do with ABI. It should use `PLATFORM_UNIX` or `PLATFORM_WINDOWS`.